modules/aerodyn/src/AeroDyn.f90

```diff
@@ -2848,6 +2848,8 @@ subroutine DiskAvgValues(p, u, m, x_hat_disk, y_hat_disk, z_hat_disk, Azimuth)
    ! calculate disk-averaged velocities
    m%AvgDiskVel = 0.0_ReKi
    m%AvgDiskVelDist = 0.0_ReKi ! TODO potentially get rid of that in the future
+   if (p%NumBlades <= 0) return  ! The Intel compiler gets array bounds issues in this routine with no blades.
+
```

Comment:
Should we make sure that all of the `intent(out)` variables in this routine have been set prior to returning? Otherwise, they may return uninitialized values, depending on compiler settings.

Reply:
Good idea.  In this case it is probably ok since none of these values really get used when there are no blades.  Still as a matter of principle it is worth doing.

I'll post another PR setting some of the uninitialized vars.

Reply:
I took a quick look in the code at where we need to set things if `p%NumBlades=0`.  This is probably going to take more time than I have at the moment, so I propose doing a PR for such changes on `dev` after the `dev-unstable` merge.